src/test/scala/org/camunda/feel/impl/builtin/BuiltinContextFunctionsTest.scala

```diff
@@ -243,6 +259,8 @@ class BuiltinContextFunctionsTest
       ValBoolean(true))
     eval(""" context(get entries({a:1,b:2})) = {a:1, b:2} """) should be(
       ValBoolean(true))
+    eval(""" context(get entries(m:{a:1,b:2})) = {a:1, b:2} """) should be(
+      ValBoolean(true))
```

Comment:
:wrench: We don't need the additional case here. 

This test case verifies the behavior of the `context()` function in combination with `get entries()`. The argument name is not relevant in this test case.